templates/chart/embed.js

```diff
@@ -1 +1,16 @@
-!function(){"use strict";window.addEventListener("message",function(a){if(void 0!==a.data["datawrapper-height"])for(var e in a.data["datawrapper-height"]){var t=document.getElementById("datawrapper-chart-"+e)||document.querySelector("iframe[src*='"+e+"']");t&&(t.style.height=a.data["datawrapper-height"][e]+"px")}})}();
+(function() {
+    'use strict';
+
+    window.addEventListener('message', function(event) {
+        if (typeof event.data['datawrapper-height'] !== 'undefined') {
+            for (var chartId in event.data['datawrapper-height']) {
+                var iframe = document.getElementById('datawrapper-chart-' + chartId) || document.querySelector("iframe[src*='" + chartId + "']");
+
+                if (!iframe) {
+                    continue;
+                }
+                iframe.style.height = event.data['datawrapper-height'][chartId] + 'px';
+            }
+        }
+    });
+})();
```

Comment:
this file needs to get simplified again before being pushed to `master`. maybe it got formatted automatically by accident?

Reply:
Will make sure this is minified before we merge back into the master branch.